src/fides/api/schemas/privacy_experience.py

```diff
@@ -82,30 +97,43 @@ class ExperienceConfigCreate(ExperienceConfigSchema):
     It also establishes some fields _required_ for creation
     """
 
-    accept_button_label: str
+    regions: List[PrivacyNoticeRegion] = []
     component: ComponentType
-    description: HtmlStr
-    reject_button_label: str
-    save_button_label: str
-    title: str
+    privacy_notices: List[str] = []
+
+    @validator("regions")
+    @classmethod
+    def validate_regions(
+        cls, regions: List[PrivacyNoticeRegion]
+    ) -> List[PrivacyNoticeRegion]:
+        """Assert regions aren't duplicated."""
+        if regions and len(regions) != len(set(regions)):
+            raise ValueError("Duplicate regions found.")
+        return regions
 
     @root_validator
     def validate_attributes(cls, values: Dict[str, Any]) -> Dict[str, Any]:
         """Validate minimum set of required fields exist given the type of component"""
         component: Optional[ComponentType] = values.get("component")
```

Comment:
We need to figure out what fields are required when adding these new translations -